modules/virt-creating-configmap.adoc

```diff
@@ -274,37 +274,23 @@ data:
     "<rhv-vm-windows-os>": "windows-template" <2>
 EOF
 ----
-<1> Optional: You can change the value of the `namespace` parameter.
+<1> Optional: You can change the namespace.
 <2> Specify the REST API name of the RHV VM Windows operating system, for example, `"windows_2016x64"`, and the Windows template.
 endif::[]
 
-. Edit the `kubevirt-hyperconverged-operator.v{HCOVersion}.yaml` file:
+. Patch the `vm-import-controller` deployment to apply the new ConfigMap:
 +
-[source,terminal,subs="attributes+"]
-----
-$ oc edit clusterserviceversion -n openshift-cnv kubevirt-hyperconverged-operator.v{HCOVersion}
-----
-
-. Update the following parameters of the `vm-import-operator` deployment manifest:
-+
-[source,yaml]
+[source,terminal]
 ----
-            ...
-            spec:
-              containers:
-              - env:
-                ...
-                - name: OS_CONFIGMAP_NAME
-                  value: os-configmap <1>
-                - name: OS_CONFIGMAP_NAMESPACE
-                  value: default <2>
+$ TARGET_NAMESPACE=openshift
+kubectl patch configmap vm-import-controller-config -n $TARGET_NAMESPACE --patch '{
+    "data": {
+        "osConfigMap.name": "os-configmap",
+        "osConfigMap.namespace": "default" <1>
+    }
+}'
 ----
-<1> Add `value: os-configmap` to the `name: OS_CONFIGMAP_NAME` parameter.
-<2> Optional: You can add this value if you changed the namespace from `default`.
-
-. Save the `kubevirt-hyperconverged-operator.v{HCOVersion}.yaml` file.
-+
-Updating the `vm-import-operator` deployment updates the `vm-import-controller` ConfigMap.
+<1> Update the namespace if you changed it in the ConfigMap.
```

Comment:
Same for the name

Reply:
As I said earlier, why bother telling them they can change the name? They will just have to update it again. 

Maybe we should do the same for the namespace -- leave it as 'default' if there is no good reason for changing it. What do you think?